client/src/components/Navbar.js

```diff
@@ -2,15 +2,51 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Nav, Navbar, NavItem, NavDropdown, MenuItem } from 'react-bootstrap';
 import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { logSignerOut } from '../redux/actions/signature';
 
-const NavBar = props => {
-  let homeText;
-  if (props.location.pathname === '/') {
-    homeText = 'RE:IMAGINE DENVER';
-  } else {
-    homeText = 'HOME';
+// RENDERS MyCampaignNavItem BASED ON AUTH STATUS
+function MyCampaignNavItem(props) {
+  return (
+    // if logged in and signed a campaign, show MyCampaign
```

Comment:
remove this comment. MyCampaignNavItem has no conception of 'logged in' means or 'campaigns' are